pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1515,20 +1562,19 @@
       ::  if conversion succeeds, we *can* send it. if the client is actually
       ::  connected, we *will* send it immediately.
```

Comment:
This comment (incl. the line above what I could select here) should probably be removed.